lib/c-api/tests/wasmer-c-api-test-runner/src/lib.rs

```diff
@@ -23,52 +25,7 @@ impl Config {
         if config.wasmer_dir.is_empty() {
             println!("manifest dir = {manifest_dir}, wasmer root dir = {wasmer_base_dir}");
             config.wasmer_dir = wasmer_base_dir.clone() + "/package";
-            if !std::path::Path::new(&config.wasmer_dir).exists() {
-                println!("running make build-capi...");
-                // run make build-capi
-                let mut cmd = std::process::Command::new("make");
-                cmd.arg("build-capi");
-                cmd.current_dir(wasmer_base_dir.clone());
-                let result = cmd.output();
-                println!("make build-capi: {result:#?}");
-
-                println!("running make package...");
-                // run make package-capi
-                let mut cmd = std::process::Command::new("make");
-                cmd.arg("package-capi");
-                cmd.current_dir(wasmer_base_dir.clone());
-                let result = cmd.output();
-                make_package();
-                println!("make package: {result:#?}");
-
-                println!("list {}", config.wasmer_dir);
-                match std::fs::read_dir(&config.wasmer_dir) {
-                    Ok(o) => {
-                        for entry in o {
-                            let entry = entry.unwrap();
-                            let path = entry.path();
-                            println!("    {:?}", path.file_name());
-                        }
-                    }
-                    Err(e) => {
-                        println!("error in reading config.wasmer_dir: {e}");
-                    }
-                };
-
-                println!("list {}/include", config.wasmer_dir);
-                match std::fs::read_dir(&format!("{}/include", config.wasmer_dir)) {
-                    Ok(o) => {
-                        for entry in o {
-                            let entry = entry.unwrap();
-                            let path = entry.path();
-                            println!("    {:?}", path.file_name());
-                        }
-                    }
-                    Err(e) => {
-                        println!("error in reading config.wasmer_dir: {e}");
-                    }
-                };
-            }
```

Comment:
Why is this code gone?

Reply:
It was running `make package` manually as a hack. I removed that in favor of running `make package && make test-integration-cli` instead.